src/Ci/AwsCodeBuild.php

```diff
@@ -0,0 +1,57 @@
+<?php declare(strict_types=1);
+
+namespace OndraM\CiDetector\Ci;
+
+use OndraM\CiDetector\CiDetector;
+use OndraM\CiDetector\Env;
+use OndraM\CiDetector\TrinaryLogic;
+
+class AwsCodeBuild extends AbstractCi
+{
+    public static function isDetected(Env $env): bool
+    {
+        return $env->get('CODEBUILD_CI') !== false;
+    }
+
+    public function getCiName(): string
+    {
+        return CiDetector::CI_AWS_CODEBUILD;
+    }
+
+    public function isPullRequest(): TrinaryLogic
+    {
+        return TrinaryLogic::createFromBoolean(mb_substr($this->env->getString('CODEBUILD_WEBHOOK_EVENT'), 0, mb_strlen('PULL_REQUEST')) === 'PULL_REQUEST');
+    }
+
+    public function getBuildNumber(): string
+    {
+        return $this->env->getString('CODEBUILD_BUILD_ID');
```

Comment:
Would `CODEBUILD_BUILD_NUMBER` be more suitable? 

`CODEBUILD_BUILD_ID` is more like "unique build identifier".

Reply:
There is no another identifier 🤷

Reply:
What do you mean? :)

CODEBUILD_BUILD_NUMBER has value like "559" etc. which is more "build number" than  values like "project-api-pr:cf59169d-ddc7-4292-a5ab-06e6f5b8c8c4" which are in CODEBUILD_BUILD_ID. 

I see build number more like human readable number of the build, not unique identifier.

Using the number is also more consistent with other CI - eg. in Travis it is used "TRAVIS_JOB_NUMBER" (with values like "25.2") and not "TRAVIS_BUILD_ID" (with values like "148395135").

Reply:
🤦 You're right 👍